screwdriver.yaml

```diff
@@ -46,8 +46,7 @@ jobs:
           yum install -y python36-pip
           pip3 install -qqq --upgrade pip
           pip3 install -qqq -r test/requirements.txt --user
-          yum install -y openssl
-          yum install -y yum-utils
+          yum install -y unzip openssl yum-utils
```

Comment:
Please add `--setopt=skip_missing_names_on_install=False`  when specifying more than one package to install with yum.